sunpy/time/time.py

```diff
@@ -71,7 +71,7 @@
 _ONE_DAY_TIMEDELTA = TimeDelta(1 * u.day)
 
 
-def is_time_equal(t1, t2):
+def is_time_equal(t1 :u.nanosecond, t2 :u.nanosecond) -> bool:
```

Comment:
Are these always nanoseconds?

Reply:
Hmm... true I need to consider multiple registers and make methods for those